### PR TITLE
[FE] Login - Logout - Register - View 

### PR DESCRIPTION
**Pull request by DTHNhung**
## Related Tickets:

-   [#45911](https://edu-redmine.sun-asterisk.vn/projects/ecommerce-intern-hn/issues)

## WHAT this PR does?

- view login logout register

## Notes Impacted Areas:

-   N/A.

## Attachments:

-   N/A.

### CODE REVIEW

**Comment by khieutrung12** [comment]
ok nhá

**Comment by DTHNhung** [comment]
ready

File: resources/views/layouts/app.blade.php [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+    
+    <!-- Styles -->
+    <link href="{{ mix('css/app.css') }}" rel="stylesheet">
+</head>
+<body class="bg-gray-100 h-screen antialiased leading-none font-sans">
+    <div id="app">
+        <header class="bg-blue-900 py-6">
+            <div class=" mx-auto flex justify-between items-center px-6">
+                <div>
+                    <a href="{{ url('/') }}" class="text-lg font-semibold text-gray-100 no-underline">
```
Comment (huyenttm-0901) [inline resources/views/layouts/app.blade.php]: ko dùng url

**Comment by DTHNhung** [comment]
ready

File: resources/views/home.blade.php [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+        <section class="flex flex-col break-words bg-white sm:border-1 sm:rounded-md sm:shadow-lg">
+
+            <header class="font-semibold bg-gray-200 text-gray-700 py-5 px-6 sm:py-6 sm:px-8 sm:rounded-t-md">
+                Dashboard
+            </header>
+
+            <div class="w-full p-6">
+                <p class="text-gray-700">
+                    You are logged in!
```
Comment (ghost) [inline resources/views/home.blade.php]: localization

File: resources/views/auth/login.blade.php [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+                                {{ old('remember') ? 'checked' : '' }}>
+                            <span class="ml-2">{{ __('titles.remember-me') }}</span>
+                        </label>
+
+                        {{-- @if (Route::has('password.request'))
+                        <a class="text-sm text-blue-500 hover:text-blue-700 whitespace-no-wrap no-underline hover:underline ml-auto"
+                            href="{{ route('password.request') }}">
+                            {{ __('messages.forgot-pass?') }}
+                        </a>
+                        @endif --}}
```
Comment (ghost) [inline resources/views/auth/login.blade.php]: ko dùng xóa lun đi bn

Comment (DTHNhung) [reply resources/views/auth/login.blade.php]: dùng vào project 2 thì có cần xóa không anh?

**Review by khieutrung12 — APPROVED** [review]

**Comment by DTHNhung** [comment]
ready

**Review by huyenttm-0901 — APPROVED** [review]

**Pull request merged**